charters/wot-ig-2021-proposed.html

```diff
@@ -370,8 +370,7 @@ <h2>Coordination</h2>
            Invitation for review must be issued during each major standards-track document transition, 
            including <a href="https://www.w3.org/2019/Process-20190301/#RecsWD" title="First Public Working Draft">FPWD</a> .
            The Interest Group is encouraged to engage collaboratively with the horizontal review groups throughout development of
-           each specification.  The Interest Group is advised to seek a review at least 3 months before first entering
-           <a href="https://www.w3.org/Consortium/Process/#RecsCR" title="Candidate Recommendation">CR</a> and is encouraged
+            each document.  The Interest Group is also encouraged
            to proactively notify the horizontal review groups when major changes occur in a specification following a review.
```

Comment:
suggest s/a specification/its technical reports/

Reply:
I've applied that change, Thanks, @swickr !
Please note that I've fixed the typo of ECLASS' name as well based on [Sebastian's comment](https://github.com/w3c/wot/pull/1006#issuecomment-971396646).

Reply:
I see no additional edits or comments to make, but I have reviewed.